OptimizelySDK/Config/FallbackProjectConfigManager.cs

```diff
@@ -0,0 +1,34 @@
+﻿/* 
+ * Copyright 2019, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OptimizelySDK.Config
+{
+    public class FallbackProjectConfigManager : ProjectConfigManager
```

Comment:
Is there gonna be more logic for the "fallback"? I think you mentioned you'd have that in a separate PR. In that case just add a `//TODO` here for it

Reply:
nope, no more logic, in other PR, I will remov SetConfig, and ProjectConfig will only be set by constructor.